Draft/draft-ietf-rtcweb-stun-consent-freshness.xml

```diff
@@ -180,13 +182,13 @@
       consent to send using ICE. Verification of peer consent before sending
       traffic is necessary in deployments like WebRTC to ensure that a
       malicious JavaScript cannot use the browser as a platform for launching
-      attacks.Section 4.4 and section 5.3 of <xref
-      target="I-D.ietf-rtcweb-security-arch"></xref> explains why webRTC
-      application needs consent.</t>
+      attacks. Section 4.4 and section 5.3 of <xref
+      target="I-D.ietf-rtcweb-security-arch"></xref> explain why webRTC
+      complaint browser must implement consent.</t>
```

Comment:
Lowercase "must".  And s/explain/explains/.  And capitalize "section".  Also, WebRTC.

> Sections 4.4 and 5.3 of <xref ... /> further explains the value of obtaining and maintaining consent.